components/ScrollUpButton/index.jsx

```diff
@@ -15,7 +15,7 @@ const ScrollUpButton = () => {
   }, []);
 
   const scrollTop = () => {
-    window.scrollTo({
+    window.scroll({
```

Comment:
Why did the scroll method change?

Reply:
It was on the testing process and ended up on the final PR
since it doesn't change the final code, I saw no reason to change back